server/models/games-models.js

```diff
@@ -93,7 +93,19 @@ exports.selectCommentsByReviewId = async (review_id) => {
      WHERE review_id = $1`,
     [review_id]
   );
-  
 
   return comments.rows;
 };
+exports.postNewComment = (review_id, username, body) => {
+  return connection
+    .query(
+      `INSERT INTO comments 
+  (author, review_id, body)
+  VALUES
+  ('${username}', ${review_id}, '${body}') 
```

Comment:
You are open to SQL injection here, use $1, $2, $3 and put the variables in an array after the query ends on line 106